app/Http/Controllers/FishRecordController.php

```diff
@@ -64,7 +64,14 @@ public function delete(Request $request, $id)
     public function detalis($id)
     {
         $rec = Fish_Record::find($id);
-        return view('Fish_Record_detalis', ['rec' => $rec]);
+        //コメントの取得
+        $comments=new Comment;
+        $fish_id=Comment::where('fish_records_id',$id);
+        $user_id=Comment::where('user_id');
+        $user_name=$comments->users()->where('name',$user_id);
+        $comments=$comments->fish_records_id=$fish_id;
+        $comments=$comments->get();
+        return view('Fish_Record_detalis', ['rec' => $rec,'comments'=>$comments,'user_name'=>$user_name]);
```

Comment:
ここはもとのコードでOKなので戻しちゃいましょう🐱
```suggestion
　　　　return view('Fish_Record_detalis', ['rec' => $rec]);
```